awx/main/migrations/0186_drop_django_taggit.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def delete_taggit_contenttypes(apps, schema_editor):
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    ContentType.objects.filter(app_label='taggit').delete()
+
+
+def delete_taggit_migration_records(apps, schema_editor):
+    from django.db.migrations.recorder import MigrationRecorder
```

Comment:
Since you've already imported the module at the top, you can get rid of this line and reference `migrations.recorder.MigrationRecorder`. Sometimes you can't do this, but I tested it with python shell and it works.